hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -285,6 +285,12 @@ name:
       en_run_modes: ["build_otbn_rig_binary_mode"]
       reseed: 5
     }
+    {
+      name: "otbn_ctrl_redun"
+      uvm_test_seq: "otbn_ctrl_redun_vseq"
+      en_run_modes: ["build_otbn_rig_binary_mode"]
+      reseed: 5
```

Comment:
Would it make sense to set the default reseed value to as many test variants as there are (12 if I counted correctly), so that every variant has a reasonable chance of getting tested?